ingest/sqs/pools/transformer/pool_transformer.go

```diff
@@ -427,3 +473,58 @@ func (pi *poolTransformer) computeUSDCPoolLiquidityCapFromUOSMO(ctx sdk.Context,
 
 	return poolLiquidityCapUOSMO, noPoolLiquidityCapError
 }
+
+func (pi *poolTransformer) updateAlloyTrasmuterInfo(
```

Comment:
Is there a way we could unit test this?

Also, would appreciate a short method spec 🙏